jspoon/src/main/java/pl/droidsonroids/jspoon/Utils.java

```diff
@@ -12,9 +14,10 @@
 import pl.droidsonroids.jspoon.exception.ConstructorNotFoundException;
 import pl.droidsonroids.jspoon.exception.ObjectCreationException;
 
-class Utils {
+public class Utils {
```

Comment:
`Utils` class shouldn't be public as it's visible for all library users

Reply:
Done.